src/mmap.cpp

```diff
@@ -111,27 +109,24 @@ mmap_reads(off_t partial_file_size,
       else
       {
         // count the number of strings in the file section
-        for ( int64_t i = reads_offset_f; i < partial_file_size-reads_offset_e-2; i++ ) if ( raw[i] == '>' ) strs++;
-
+        for ( int64_t i = reads_offset_f; i < partial_file_size-reads_offset_e-2; i++ ) if ( raw[i] == '>' )
```

Comment:
The significance of ```-2``` is to remove the "\n>" characters from search at the end of the mapped file section. I've added this in the comment.

for the code:
```
for ( int64_t i = reads_offset_f; i < partial_file_size-reads_offset_e-2; i++ )
```

we have:
```
file section 1
>seq1\n
TGTGA\n
    ^ (2) the -2 reduces the search space to here
>seq2\n
^ (1) reads_offset_e sets the search space to here
AA ..
```